CHANGES.md

```diff
@@ -41,6 +41,7 @@
 - Avoid copying unnecessarily large amounts of strings when parsing (#85, #108,
   @Leonidas-from-XIV)
 - Fix `stream_to_file` (#133, @tcoopman and @gasche)
+- Fix pretty-printing behavior for lists/objects inside lists
```

Comment:
This should actually go into a new `unreleased` section, I'll push this on `master`.